packs/mailgun/actions/send_email.py

```diff
@@ -0,0 +1,31 @@
+import httplib
+
+import requests
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'SendEmailAction'
+]
+
+SEND_EMAIL_API_URL = 'https://api.mailgun.net/v2/%(domain)s/messages'
+
+
+class SendEmailAction(Action):
```

Comment:
Class names are local to the pack and the file so explicit namespacing is not necessary - as long as the class is in a different file and / or pack, there should be no problems.

Same goes for sensors, although sensors are a bit different - they explicitly specify class name in the metadata file, where in the python-runner action case, we just pull out the first class from the file (I'd actually like to make this more explicit and consistent with the sensor, eventually...).